src/containers/MyNdla/Folders/DraggableResource.tsx

```diff
@@ -152,6 +157,26 @@ const DraggableResource = ({
     transition,
   };
 
+  const [resourceType, resourcePath, resourceTitle] = useMemo(() => {
+    let resType: GQLFolderResourceResourceType[] = [];
+    let resPath = resource.path;
+
+    if (!resourceMeta) {
+      return [resType, resPath, t("myNdla.sharedFolder.resourceRemovedTitle")];
+    }
+
+    if (resourceMeta.resourceTypes && resourceMeta.resourceTypes.length > 0) {
+      resType = resourceMeta.resourceTypes;
+    } else {
+      resType = [{ id: resource.resourceType, name: t(`contentTypes.${resource.resourceType}`) }];
+      if (resource.resourceType === "article") {
+        resPath = `/article/${resource.resourceId}`;
+      }
+    }
+
+    return [resType, resPath, resourceMeta.title];
+  }, [resource, resourceMeta, t]);
```

Comment:
Trenger dette å være muterbart? Tror jeg hadde foretrukket early returns.

Reply:
Var egentlig mest et forsøk på å sjekke resourceMeta kun 1 gang for de 3 attributtene som kan påvirkes av den (og da veit jeg ikke om jeg kan returnere oftere enn de 2 stedene jeg har satt). Ikke sikkert det er særlig (if any) å hente på det, så kan godt splitte det opp igjen.